x-pack/plugins/osquery/public/results/results_table.tsx

```diff
@@ -216,29 +217,51 @@ const ResultsTableComponent: React.FC<ResultsTableComponentProps> = ({
     [actionId, endDate, startDate]
   );
 
+  useEffect(
+    () =>
+      setIsLive(
+        aggregations.totalResponded !== agentIds?.length ||
+          // @ts-expect-error-type
+          allResultsData?.rawResponse.aggregations?.unique_agents.value !== agentIds?.length
```

Comment:
will this do the right thing when either side is `null`? should this be flagged as live in the situation where there are no aggregations returned, or the agent ids array is missing?